### ISSUE TITLE
curly braces are not fixed

### ISSUE THREAD

**Issue by MikhailLipanin** [yegor256/bibcop#8]
I have this `main.bib`:
```
@misc{joule,
  author     = {Tribble, Dean E. and Miller, Mark S. and Hardy, Norm and Krieger, David},
  title      = {{Joule: Distributed Application Foundations. Technical Report {ADd03.4P}}},
  year       = 1995,
  publisher  = {Agorics Inc.},
}
```
after applying `bibcop`, it replaces with this:
```
@misc{joule,
  author = {Tribble, Dean E. and Miller, Mark S. and Hardy, Norm and Krieger, David},
  publisher = {Agorics Inc.},
  title = {{Joule: Distributed Application Foundations. Technical Report {ADd03.4P}},
  year = {1995},
}
```

Note, that in title there was removed a curly brace, so it is an error

**Comment by MikhailLipanin** [comment]
@yegor256 FYI

**Comment by yegor256** [comment]
@MikhailLipanin I believe it's fixed in dba406c
